sbin/build-redisjson

```diff
@@ -54,6 +54,12 @@ $OP cd RedisJSON
 runn git checkout $BRANCH
 runn git pull --quiet --recurse-submodules
 
+# Patch RedisJSON to build in Alpine - disable static linking
+OS_NAME=$(cat /etc/os-release | egrep "^NAME=" | cut -d '=' -f 2 | tr -d '"')
```

Comment:
Consider using a common way to get the OS name (as done in other scripts using sed, etc.)